core/trino-main/src/main/java/io/trino/execution/buffer/OutputBufferMemoryManager.java

```diff
@@ -199,6 +229,7 @@ public synchronized void close()
             memoryContext.close();
         }
         closed = true;
+        finalOutputBufferUtilization.set(new TDigestHistogram(TDigest.copyOf(outputBufferUtilization)));
```

Comment:
This (`finalOutputBufferUtilization`) won't be propagated via final `TaskInfo`, see https://github.com/trinodb/trino/pull/13463#discussion_r952373652